app/src/main/java/de/westnordost/streetcomplete/quests/surface/AddRoadSurface.kt

```diff
@@ -24,7 +27,7 @@ class AddRoadSurface : OsmFilterQuestType<SurfaceAnswer>() {
           or surface ~ ${ANYTHING_UNPAVED.joinToString("|")} and surface older today -6 years
           or surface older today -12 years
           or (
-            surface ~ paved|unpaved|cobblestone
+            surface ~ paved|unpaved|${Surface.invalidSurfaces.joinToString("|")}
```

Comment:
Hu, should it not be INVALID_SURFACES? This (companion object?) looks very Java-y